src/core/core.animation.js

```diff
@@ -42,6 +42,19 @@ export default class Animation {
 		return this._active;
 	}
 
+	update(cfg, to, date) {
+		const me = this;
+		if (me._active) {
+			const currentValue = me._target[me._prop];
+			const elapsed = date - me._start;
```

Comment:
I don't know that this is guaranteed to work in 100% of cases. `Date.now()` is not guaranteed to be monotonic. https://stackoverflow.com/questions/7272395/monotonically-increasing-time-in-javascript

Reply:
True, it could go wrong if the animation is running when going to/from DST or altering system time. The ticking is also based on Date.now(), do I don't think this makes it  any worse.

Reply:
Fair, and if its wrong and someone refreshes, it's not the end of the world. Might be nice to try and replace `Date.now()` everywhere, but I'm not sure what the correct solution in